### DISCUSSION TITLE
Rust vs python versioning

### DISCUSSION THREAD

**Discussion by iiLaurens** [delta-io/delta-rs#1547] [Q&A] [answered]
I am slightly confused about the versioning between the rust and python releases. I monitor the changes to see what features are supported and issues are fixed. Since the python package offloads most work to the rust libraries, I imagine a lot of fixes in rust also impact the python package.

What confuses me is the gap in versioning. Does python version v0.10.0 imply that only changes up to the rust release v0.10.0 are included? Do I have to be careful that all bugfixes in rust package >v0.10.0 are still present in python package v0.10.0? E.g. given a shared version between rust and python, are the features in sync?

If they are not in sync, how can one tell which version of the rust library is used by the python library?

**Comment by wjones127** [comment, marked as answer]
I would pay attention to the Python release notes. They having everything that's relevent.

You can get the Rust version used in python with `deltalake.rust_core_version`
